managers/kafka/handler.py

```diff
@@ -0,0 +1,95 @@
+""" The implementation class for KafkaManager, the public interface used by main. """
+
+import asyncio
+
+from aiokafka.errors import KafkaError
+
+from kytos.core import log
+from napps.kytos.kafka_events.managers.kafka._producer import Producer
+from napps.kytos.kafka_events.managers.kafka._serializer import JSONSerializer
+from napps.kytos.kafka_events.managers.kafka.interface import KafkaManager
+from napps.kytos.kafka_events.settings import (
+    ACKS,
+    BATCH_SIZE,
+    BOOTSTRAP_SERVERS,
+    ENABLE_ITEMPOTENCE,
+    LINGER_MS,
+    MAX_REQUEST_SIZE,
+    TOPIC_NAME,
+)
+
+
+class KafkaDomainManager(KafkaManager):
+    """Acts like an orchestrator for internal components."""
+
+    def __init__(self):
+        """
+        Object-oriented, with separate classes for specific tasks
+        """
+        self._producer = Producer(
+            bootstrap_servers=BOOTSTRAP_SERVERS,
+            acks=ACKS,
+            enable_itempotence=ENABLE_ITEMPOTENCE,
+            topic_name=TOPIC_NAME,
+            max_batch_size=BATCH_SIZE,
+            linger_ms=LINGER_MS,
+            max_request_size=MAX_REQUEST_SIZE,
+        )
+        self._serializer = JSONSerializer()
+
+    async def send(self, event: str, message: any) -> None:
+        """
+        Send data to Kafka. Uses the following flow:
+
+        - Checks that the producer was not closed
+        - Checks that the producer is ready
+        - Serializes the message into JSON
+        - Awaits the producer to enqueue the message
+        """
+        try:
+            await self._producer.send_data(
+                self._serializer.serialize_and_encode(event, message)
+            )
+        except asyncio.TimeoutError:
+            log.error("Producer tried publishing data but timed out.")
```

Comment:
We'll still have tenacity retries, but imagine when `TimeoutError` happens, how will we know which event did timeout for correlation to try to understand what happened? We should consider at least including the event name, and its id too, and probably with its timestamp too. 

Which also bring the point that maybe passing down `KytosEvent` instance instead of `event: str` and `message: any` would be more appropriate as less likely to need refactoring later on. Think about it.